addons/rating_project/views/project_view.xml

```diff
@@ -23,6 +23,9 @@
                     <separator />
                     <filter string="Rating for tasks" name="rating_tasks" domain="[('res_model', '=', 'project.task')]" />
                 </xpath>
+                <xpath expr="//filter[@name='Responsible']" position="before">
```

Comment:
Also make all inherited views in rating_project `primary`. Since rating is not only used in project, we don't want to see 'Rating for task' when we are looking at livechat rating.
Adapt ir.actions accordingly.